commit-msys2.sh

```diff
@@ -170,7 +170,7 @@ commit)
 	 git add -A . &&
 	 if git diff-index --exit-code --cached HEAD -- \
 		':(exclude)var/lib/pacman/sync/' \
-		':(exclude)var/lib/pacman/local/git-extra-*/desc' \
+		':(exclude)var/lib/pacman/local/mingw-w64-*-git-extra-*/desc' \
```

Comment:
Here, too, I would like to accept both old _and_ new name.

```suggestion
		':(exclude)var/lib/pacman/local/git-extra-*/desc' \
		':(exclude)var/lib/pacman/local/mingw-w64-*-git-extra-*/desc' \
```